contracts/bounty/RexBounty.sol

```diff
@@ -5,18 +5,13 @@ import "hardhat/console.sol";
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
-import "../tellor/ITellor.sol";
 import "@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol";
 import "@openzeppelin/contracts/utils/Counters.sol";
-
-import { SuperAppBase } from "@superfluid-finance/ethereum-contracts/contracts/apps/SuperAppBase.sol";
```

Comment:
Why are we removing `superAppBase` here?

Reply:
nm, The captain doesn't need to be a super app and I think we agreed to shift bounty payments to `RexBounty`.